test/e2e/scalability/density.go

```diff
@@ -768,6 +772,9 @@ var _ = SIGDescribe("Density", func() {
 					framework.ExpectNoError(framework.DeleteRCAndWaitForGC(c, rcNameToNsMap[name], name))
 				}
 				workqueue.Parallelize(25, nodeCount, deleteRC)
+
+				// Fail if more than the allowed threshold of measurements were missing.
+				Expect(missingMeasurements <= MaxMissingPodStartupMeasurements).To(Equal(true))
```

Comment:
Can we for now set "MaxMissing..." to 0?

And to avoid this affecting the test itself, call it at the end of "AfterEach" for the test?

Reply:
Changed it to 0.
I think we should be making this check this block of code itself as it's only relevant to runLatencyTest. And I moved it to the end precisely because of this (to not affect other parts of the test).
And `AfterEach` runs always (even if the test fails) - so that should be fine.

Reply:
But we won't call "cleanupDensityTest" if that fails. And that is pretty crucial for the whole test itself.

Reply:
Makes sense.. Moved it to AfterEach.

Reply:
It's not moved.

Reply:
Sorry, I pushed the change to wrong branch. PTAL now.